ux.symfony.com/templates/ux_packages/map.html.twig

```diff
@@ -3,15 +3,18 @@
 {% set package = 'map'|ux_package %}
 
 {% block package_header %}
-    <twig:Hero
-        eyebrows="Assert('Mapper')"
-        title="UX Map"
-        url="{{ url('app_map') }}"
-    >
-        Embed <code>interactive maps</code> in your Symfony application in a breeze!
-        <br />
-        Decouple your code from your map provider: <code>Google Maps</code>, <code>LeaftLet</code>.
-    </twig:Hero>
+    {% component PackageHeader with {
+        package: 'map',
+        eyebrowText: 'Seamless Maps Integration',
+    } %}
+        {% block title_header %}
+            Embed <em>interactive maps</em> in a breeze!
+        {% endblock %}
+
+        {% block sub_content %}
+            Decouple your code from your map provider: <em>Google Maps</em>, <em>LeaftLet</em>.
```

Comment:
I don't really understand this syntax what do you think about something like: Seamlessly connect with any map provider, like Google Maps or Leaflet.

Reply:
_Seamless_ is used in the eyebrow, and _decouple_ felt a good thing to expose as it is an important promise of the component, but i agree this sentence could be more catchy / efficient